README.md

```diff
@@ -1 +1,54 @@
-# plugin_sublime
\ No newline at end of file
+![Pieces Sublime Plugin](https://camo.githubusercontent.com/69c990240f877927146712d45be2f690085b9e45b4420736aa373917f8e0b2c8/68747470733a2f2f73746f726167652e676f6f676c65617069732e636f6d2f7069656365735f7374617469635f7265736f75726365732f7066645f77696b692f5049454345535f4d41494e5f4c4f474f5f57494b492e706e67)
+
+
+##### <p align="center"> [Website](https://pieces.app/) • [Pieces OS Documentation](https://docs.pieces.app/) • [Pieces Sublime github repository](https://github.com/pieces-app/plugin_sublime)
+</p>
+
+
+## Getting Started
+
+Welcome to the Pieces Sublime Plugin.
+Pieces Sublime plugin robust plugin designed to manage your pieces snippets and interact seamlessly with the Pieces OS client.
+To get started, navigate to `Preferences > Package Settings > Pieces > Keybindings` to check out and modify the keybindings. If you wish to change the llm model you're using or the host, you can do so in the settings.
+
+### Open Asset Command
+
+To open your assets, simply open your command palette and run `Pieces: Open an asset`. Then, select the asset you wish to open.
+
+#### Editor Mode
+
+Enter the editor mode with `command/ctrl + e`. You can then modify your asset and save it using `command/ctrl + s`.
+
+#### Copy an Asset
+
+If you're in an asset sheet, you can quickly copy the asset content without selecting it using `command/ctrl + c`.
+
+### Copilot Ask
+
+Need the copilot to comment, fix a bug, or add a doc string? Simply go to the command palette and run `Pieces: Ask Copilot`.
+
+### Login/Logout
+
+You can log in and out of your Pieces account using the `Pieces: Login` or `Pieces: Logout` command in your command palette. An output panel is available to check your cloud status and login status.
+
+### Right click to add an asset
+
+You can create a new asset to pieces select the text, right click then choose `pieces>save to pieces`.
```

Comment:
"To save a new snippet in Pieces, select the code you wish to save, right-click to open the context menu, and select "Pieces > Save to Pieces".